developer/guides/deploying/overview.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Overview
+description: "Learn how to deploy your Makeswift application."
+---
+
+## Supported hosting platforms
+
+<CardGroup cols={3}>
+  <Card
+    title="Vercel"
+    icon="lightbulb"
```

Comment:
Did we want to pick a different logo for each or do we not have good options available?

Reply:
I think removing the logo for now makes sense. I can't think of a way we would want to distinguish them other than their own logos but  I don't thikn that's an option with Mintlify

Reply:
Sounds good 👍🏻